config/jobs/kubernetes/node-problem-detector/node-problem-detector-ci.yaml

```diff
@@ -0,0 +1,135 @@
+periodics:
+- name: ci-npd-build
+  interval: 2h
+  decorate: true
+  labels:
+    preset-service-account: "true"
+  extra_refs:
+  - org: kubernetes
+    repo: node-problem-detector
+    base_ref: master
+    path_alias: k8s.io/node-problem-detector
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20190220-615bdbe2e-master
```

Comment:
@krzyzacy What image should I use here? I copied from other config files. Is this tied to a particular k8s version?

Reply:
depend on if you need a specific go and bazel version - you can see the difference in https://github.com/kubernetes/test-infra/blob/master/images/kubekins-e2e/Makefile#L28-L64

Reply:
ACK